### PR TITLE
Update v7-feature-tracker.md

### PR DESCRIPTION
**Pull request by RexJaeschke**
Reflects the decisions made on the 2022-11-30 TC49/TG2 telcon.

### CODE REVIEW

File: admin/v7-feature-tracker.md [hunk 1]
```diff
@@ -12,21 +12,21 @@ Version | Feature | PR | Status | Effort | Notes
...
+7.0 | `out` variables ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.0/out-var.md)) | [664](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/664), which replaced [44](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/44) | Open | Small | Feature Group B.
+7.0 | Discards ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.0/out-var.md)) | [664](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/664), which replaced [44](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/44) and [596](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/596) | Open | Small | Feature Group B.
+7.0 | Tuples ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.0/tuples.md)) | [63](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/63) **but needs some work**, [664](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/664), which replaced [596](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/596) | Open | Large | Feature Group C.
```
Comment (jskeet) [inline admin/v7-feature-tracker.md]: I suspect 664 will replace 63 as well.

File: admin/v7-feature-tracker.md [hunk 2]
```diff
@@ -39,6 +39,6 @@ Version | Feature | PR | Status | Effort | Notes
 7.3 | Support for Pattern-Based `fixed` Statements ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.3/pattern-based-fixed.md)) | [240](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/240) | Open | Small |  
 7.3 | use additional generic constraints ([MS Proposal 1](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.3/blittable.md)) ([MS proposal 2](https://github.com/dotnet/csharplang/issues/104)) ([MS proposal 3](https://github.com/dotnet/csharplang/issues/103)) | [244](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/244) | Open | Small | 
 7.3 | test `==` and `!=` with tuple types ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.3/tuple-equality.md)) | [63](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/63) **but needs some work** | Open | Large | Feature Group C.
```
Comment (jskeet) [inline admin/v7-feature-tracker.md]: I suspect 664 may handle this too.

File: admin/v7-feature-tracker.md [hunk 3]
```diff
@@ -12,21 +12,21 @@ Version | Feature | PR | Status | Effort | Notes
...
 7.1 | `async Main` method ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/async-main.md)) | [70](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/70) | Completed | Small | 
-7.1 | `default` literal expressions ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/target-typed-default.md)) | [236](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/236) | Open | Small | 
+7.1 | `default` literal expressions ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/target-typed-default.md)) | [236](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/236) | Completed | Small | 
 7.1 | Inferred tuple element names ([MS Proposal](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-7.1/infer-tuple-names.md)) | [63](https://github.com/ECMA-TC49-TG2/csharpstandard/pull/63) **but needs some work** | Open | Large | Feature Group C.
```
Comment (jskeet) [inline admin/v7-feature-tracker.md]: Update this for 664?

**Review by BillWagner — COMMENTED** [review]
I have no comments beyond Jon's.

Once addressed, let's :shipit:

File: admin/v7-feature-tracker.md (hunk 1 above)
Comment (RexJaeschke) [reply admin/v7-feature-tracker.md]: Updated as suggested.

File: admin/v7-feature-tracker.md (hunk 2 above)
Comment (RexJaeschke) [reply admin/v7-feature-tracker.md]: Changed as suggested.

File: admin/v7-feature-tracker.md (hunk 3 above)
Comment (RexJaeschke) [reply admin/v7-feature-tracker.md]: Updated as suggested.

**Comment by RexJaeschke** [comment]
@BillWagner Please merge this.

**Review by BillWagner — APPROVED** [review]
LGTM.

**Pull request merged**